### PR TITLE
Add Nexii to list of blacklisted devices, add secondary option for theme only disabling, rollback to Unity 2.2

### PR DESCRIPTION
**Pull request by lzlrd**
Thx to @ThatNerdyPikachu

### CODE REVIEW

**Comment by ThatNerdyPikachu** [comment]
Is that okay if I move some other changes from my additions to yours?

**Comment by lzlrd** [comment]
> 
> 
> Is that okay if I move some other changes from my additions to yours?

Such as?

**Comment by ThatNerdyPikachu** [comment]
Some of the wording + the Unity downgrade. (cc @Stallix)

**Comment by lzlrd** [comment]
> 
> 
> Some of the wording + the Unity downgrade. (cc @Stallix)

I'd check up with @JohnFawkes about the Unity downgrade.

**Comment by ThatNerdyPikachu** [comment]
@lazerl0rd Please review my changes, I'll check with John and Zack about the downgrade.

**Comment by lzlrd** [comment]
@ThatNerdyPikachu the user needs to understand the difference between theme removal and overlay removal.

**Comment by ThatNerdyPikachu** [comment]
will fix.

**Comment by lzlrd** [comment]
@ThatNerdyPikachu The overlay file (Pix3lify.apk) contains more features than rounded corners. It contains Night Light too, for example.

The Pixel folder in $INSTALLER/system/vendor/overlay contains the Pixel theme. The build.prop for it you know already.

So your wording needed a bit of change ;).

File: common/install.sh [hunk 1]
```diff
@@ -90,16 +90,31 @@ $FUNCTION "DOWN"
...
+ui_print "   Vol Up = Yes, Vol Down = More Options"
 if $FUNCTION; then
   ui_print " "
-  ui_print "   Enabling overlays..."
+  ui_print " - Overlay Options -"
+  ui_print "   Do you want the Pixel accent enabled?"
+  ui_print "   Vol Up = Yes, Vol Down = No"
+  if $FUNCTION; then
+    ui_print " "
+    ui_print "   Enabling overlays and Pixel blue accent..."
```
Comment (ThatNerdyPikachu) [inline common/install.sh:103]: This is supposed to disable it.

```
User wants overlay and tint?
-> Do that
Else?
-> Show a menu
    - Do they want just the overlay features?
        - Disable tint but keep overlays
    - Do they want nothing?
        - Disable everything
```

Once you fix that, I'll make my final grammar changes, as usual. (i'm too tired to fix)

**Review by ThatNerdyPikachu — CHANGES_REQUESTED** [review]
You broke it lol

File: common/install.sh (hunk 1 above)
Comment (lzlrd) [reply common/install.sh:103]: Ahh, I see.

**Comment by ThatNerdyPikachu** [comment]
Also, I cannot push to your repo, therefore I cannot upload Unity v2.2.

Can you add this in ``common``?
https://github.com/Zackptg5/Unity/blob/master/common/unityfiles.tar.gz

**Pull request merged**